srtp/crypto/openssl/hmac.cpp

```diff
@@ -75,7 +94,11 @@ void* initializeSha1HmacContext(void* ctx, uint8_t* key, int32_t keyLength)
 {
     HMAC_CTX *pctx = (HMAC_CTX*)ctx;
 
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
     HMAC_CTX_init(pctx);
+#else
+	HMAC_CTX_reset(pctx);
```

Comment:
If `pctx` a la `HmacCtx. hmacSha1Ctx` is `NULL` or not yet (pointing to [depending on the openssl version]) an `HMAC_CTX`, then `HMAC_CTX_reset(pctx)` is wrong here and `HMAC_CTX_new()` should be used instead. I'll investigate but I think that this is the case, unfortunately.